lldb/test/API/python_api/sbvalue_is_valid/main.cpp

```diff
@@ -0,0 +1,6 @@
+int main () {
+  int i = 42;
+  return 0; //% v1 = self.frame().EvaluateExpression("test"); v2 = self.frame().EvaluateExpression("i");
+  //% self.assertFalse(v1.IsValid())
+  //% self.assertTrue(v2.IsValid())
```

Comment:
Once you have more than one line of comments, a regular (non inline) python tests is IMO more readable.